### PR TITLE
fix format for `:hook` directive

### PR DESCRIPTION
**Pull request by herminiotorres**
it should not format when has an `:hook` directive without any attribute.

This fix it is related with this PR here: https://github.com/surface-ui/surface/pull/636

### CODE REVIEW

**Comment by paulstatezny** [comment]
@herminiotorres Thanks for the PR!

@msaraiva @herminiotorres I think this change needs to be added https://github.com/surface-ui/surface/pull/636 — is that correct?

File: test/surface/formatter_test.exs [hunk 1]
```diff
@@ -1000,6 +1000,12 @@ defmodule Surface.FormatterTest do
       )
     end
 
+    test ":hook directive without any attribute" do
+      assert_formatter_doesnt_change("""
+      <div :hook />
+      """)
+    end
```
Comment (paulstatezny) [inline test/surface/formatter_test.exs:1007]: This would have been great to add to the formatter in [surface-ui/surface](https://github.com/surface-ui/surface) itself :)

Care to put in another PR?

**Review by paulstatezny — APPROVED** [review]
Thanks again @herminiotorres !

File: test/surface/formatter_test.exs (hunk 1 above)
Comment (herminiotorres) [reply test/surface/formatter_test.exs:1007]: sure, I am glad to add this to another PR
